modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -493,6 +493,34 @@ $ docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
 CLEAN_PERFORMANCE_PROFILE="false" registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ----
 
+[id="cnf-performing-end-to-end-tests-running-in-single-node-cluster_{context}"]
+== Running in single node cluster
+
+Running tests on a single node cluster causes the following limitations to be imposed:
+
+* Longer timeouts for certain tests, including SR-IOV and SCTP tests
+* Tests requiring master/worker nodes are skipped
+
+Longer timeouts concern SR-IOV and SCTP tests. Reconfiguration requiring node reboots cause a reboot of the entire environment, including the OpenShift control plane, and therefore takes longer to complete. All PTP tests requiring a master and worker node are skipped. No additional configuration is needed because the tests check for the number of nodes at startup and adjust test behavior accordingly.
+
+PTP tests can run in Discovery mode. The tests will look for a PTP master configured outside of the cluster.
+
+For more information, see the Discovery mode section.
+
+To enable Discovery mode, the tests must be instructed by setting the `DISCOVERY_MODE` environment variable as follows:
+
+[source,terminal]
+----
+$ docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
+DISCOVERY_MODE=true registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
+----
+
+=== Required parameters
```

Comment:
You might want to consider adding the `[discrete]` tag above this to make this a numberless heading (since it's tied to the above section). Headings aren't numbered on docs.openshift.com, but they are on the portal. Usually if it's a heading that doesn't need to be in the TOC, we probably would use that tag. (It shouldn't show up in the TOC because it's a level 4 and we only show 3 levels in the TOC I believe, but still the same principle).